ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/LMDB.hs

```diff
@@ -365,22 +414,23 @@ newLMDBBackingStore dbTracer limits sfs init_db = do
     (gdd, copy_db_action :: m (), init_action :: Db m l -> m ()) = case init_db of
       LIInitialiseFromLMDB fp
         -- If fp == path then this is the LINoInitialise case
-        | fp /= path -> (GDDMustNotExist, initFromLMDBs dbTracer sfs fp path, \_ -> pure ())
+        | fp /= path -> (GDDMustNotExist, initFromLMDBs dbTracer limits sfs fp path, \_ -> pure ())
       LIInitialiseFromMemory slot vals -> (GDDMustNotExist, pure (), initFromVals slot vals)
       _ -> (GDDMustExist, pure (), \_ -> pure ())
 
   -- get the filepath for this db creates the directory if appropriate
-  dbFilePath <- guardDbDir gdd sfs path
+  dbFilePath <- guardDbDirWithRetry gdd sfs path
 
   -- copy from another lmdb path if appropriate
   copy_db_action
 
   -- open this database
   dbEnv <- liftIO $ LMDB.openEnvironment dbFilePath limits
 
-  -- the LMDB.Database that holds the DbState (i.e. sequence number)
+  dbSettings <- liftIO $ LMDB.readWriteTransaction dbEnv $ LMDB.getDatabase (Just "_dbsettings")
```

Comment:
Do we do anything with this? If not, how will we use this?

Reply:
Like I mentioned in a previous comment, I wonder if we could assume these settings are passed to `newLMDBBackingStore`.